.enb/helpers/page.js

```diff
@@ -0,0 +1,171 @@
+var fs = require('fs'),
+    path = require('path'),
+    config = require('../config'),
+    techs = require('../techs');
+
+/**
+ * Configures dir to build page.
+ *
+ * Before build in file system must exist directory with BEMJSON file.
+ *
+ * page/
+ * └── page.bemjson.js
+ *
+ * Important: BEMJSON file should be provided.
+ *
+ * After build we get js, CSS and HTML files.
+ *
+ * page/
+ * ├── page.bemjson.js
+ * ├── page.css
+ * ├── page.js
+ * └── page.html
+ *
+ * @param {NodeConfig} node — instance for configure dir
+ * @param {Object} opts
+ * @param {Object} opts.platform — platform name (desktop or touch).
+ * @param {String[]} [opts.langs] — languages.
+ * @example
+ * var configurePage = require('./helpers/page'),
+ *     FileProviderTech = require('enb/techs/file-provider');
+ *
+ * module.exports = function(project) {
+ *     project.node('desktop.bundles/index', function (node) {
+ *         // provide BEMJSON file
+ *         node.addTech([FileProviderTech, { target: '?.bemjson.js' }]);
+ *
+ *         // build page
+ *         configurePage(node, {
+ *             platform: 'desktop',
+ *             langs: ['en', 'ru']
+ *         });
+ *     });
+ * };
+ */
+module.exports = function(node, opts) {
+    var nodeUrl = node.getPath();
+
+    opts = opts || {};
+
+    if(!opts.platform) {
+        var error = new Error('platform to configure the `' + nodeUrl + '` node is not specified.');
+        error.name = 'ConfigError';
+
+        throw error;
+    }
+
+    var langs = opts.langs || [],
+        nodeDirname = node.getNodePath(),
+        sublevels = [
+            path.join(nodeDirname, '..', '.blocks'),
+            path.join(nodeDirname, 'blocks')
+        ].filter(function (level) {
+            return fs.existsSync(level);
+        }),
+        levels = [].concat(
+            config.levels(opts.platform),
+            sublevels
+        ),
+        needBEMTREE = opts.bemtree;
+
+    node.addTechs([
+        // get FileList
+        [techs.bem.levels, { levels : levels }],
+        !needBEMTREE && [techs.bem.bemjsonToBemdecl],
+        [techs.bem.depsOld],
+        [techs.bem.files],
+
+        // build CSS
+        [techs.css],
+
+        // build BEMHTML for browsers
+        [techs.bem.depsByTechToBemdecl, {
+            target : '?.bemhtml.bemdecl.js',
+            sourceTech : 'js',
+            destTech : 'bemhtml'
+        }],
+        [techs.bem.depsOld, {
```

Comment:
No, but this PR about refactoring only.

All improvements suggest doing separately.